server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -288,7 +288,7 @@ object InteractiveSession extends Logging {
         case (Some(file), false) =>
           debug("Enable HiveContext because hive-site.xml is found under classpath, "
             + file.getAbsolutePath)
-          mergeConfList(List(file.getAbsolutePath), LivyConf.SPARK_FILES)
+          mergeConfList(List(file.getAbsolutePath), LivyConf.SPARK_JARS)
```

Comment:
spark.files is "Comma-separated list of files to be placed in the working directory of each executor."
this mean,  hive-site.xml is not deploy with livy jars when yarn-cluster mode